src/content/docs/tutorials/sync/overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Overview
+subtitle: Learn how nf-core pipelines are automatically kept up to date with community standards.
+weight: 1
+---
+
+# Introduction
+
+To keep all the nf-core pipelines up-to-date with the latest version of the community standards, we have implemented a synchronisation tool.
+This ensures that updates to the community standards are propagated to all nf-core pipelines.
+
+There are three topics covered in this documentation page:
+
+1. [Merging automated PRs](#merging-automated-prs)
+2. [Manual synchronisation](#manual-synchronisation)
+3. [Setting up a pipeline for syncing retrospectively](#setting-up-a-pipeline-for-syncing-retrospectively)
+4. [Fixing a broken `TEMPLATE` branch](#fixing-a-broken-template-branch)
```

Comment:
These sections don't exist on this file, I think those are the right paths

```suggestion
1. [Merging automated PRs](/docs/tutorials/sync/merging_automated_prs)
2. [Manual synchronisation](/docs/tutorials/sync/manual_sync)
3. [Setting up a pipeline for syncing retrospectively](/docs/tutorials/sync/sync_retrospectively)
4. [Fixing a broken `TEMPLATE` branch](/docs/tutorials/sync/fixing_broken_template_branch)
```

Reply:
Thanks! I fear that there may be more of these, as we broke up quite a lot of pages and it's easy to leave hanging anchor links like this.

We'll just have to do our best to find and fix them as they crop up.